keyboards/inett_studio/sqx/universal/info.json

```
@@ -0,0 +1,33 @@
+{
+    "keyboard_name": "sqx",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 15,
+    "height": 5,
+    "layouts": {
+        "LAYOUT_60_wkl": {
+            "key_count":61,
```

Many of these layouts are Community Layouts, or variants thereof.

Also, `key_count` is deprecated.

```suggestion
        "LAYOUT_60_ansi_tsangan_split_rshift": {
```